### PR TITLE
Correct Synapse install command for FreeBSD.

### PR DESCRIPTION
**Pull request by itchychips**
### Pull Request Checklist

* [X] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
* [x] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
* [x] [Code style](https://matrix-org.github.io/synapse/latest/code_style.html) is correct
  (run the [linters](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#run-the-linters))

### CODE REVIEW

**Comment by itchychips** [comment]
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
660 York Street, Suite 102,
San Francisco, CA 94110 USA

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Donny Johnson <itchychips@itchyeyes.net>

**Review by DMRobertson — COMMENTED** [review]
Seems reasonable! Could you explain what changed to make this necessary?

Not familiar with the FreeBSD packaging machinery. Judging from https://bugs.freebsd.org/bugzilla/attachment.cgi?id=229255&action=diff it looks like the package name is prefixed with `PYTHON_PKGNAMEPREFIX`, which I'm guessing was set to py38 recently?

**Comment by reivilibre** [comment]
http://www.sysfault.org/posts/freebsd-python-ports-pre-suffix.html seems like a noob-friendly piece of writing on it, so I think the changes seem reasonable.

File: changelog.d/11267.doc [hunk 1]
```diff
@@ -0,0 +1 @@
+Correct synapse install command for FreeBSD.
```
Comment (reivilibre) [inline changelog.d/11267.doc:1]:
```suggestion
Update Synapse install command for FreeBSD as the package is now prefixed with `py38`. Contributed by @itchychips.
```
Noob talking here, so correct me if this sounds rubbish. Would you like to be mentioned in the changelog? :)

**Comment by richvdh** [comment]
@reivilibre @DMRobertson what info is this waiting for?

**Comment by DMRobertson** [comment]
> @reivilibre @DMRobertson what info is this waiting for?

From https://github.com/matrix-org/synapse/pull/11267#pullrequestreview-799964919:

>  Could you explain what changed to make this necessary?

Not being familiar with the FreeBSD ecosystem, I wanted to understand why this was needed/what it means before merging.

**Pull request merged**